services/request/index.js

```diff
@@ -11,24 +11,27 @@ function createRequestService() {
                 accept: 'application/vnd.api+json',
                 'Content-Type': 'application/vnd.api+json'
             },
-            json: true,
-            body: options.body
```

Comment:
This is what I was trying (poorly) to describe on Friday.  By removing the explictly set `body: options.body`, but still merging this object with `options`, means you will still have something like
```
{
  options: {
    body: <payload>
  }
} 
```

Rather than the suggested

```
{
  options: {
    json: <payload>
  }
}
```